pkg/generated/applyconfiguration/alerting_notifications/v0alpha1/integration.go

```diff
@@ -5,20 +5,20 @@
 package v0alpha1
 
 import (
-	json "encoding/json"
+	v0alpha1 "github.com/grafana/grafana/pkg/apimachinery/apis/common/v0alpha1"
 )
 
-// IntegrationApplyConfiguration represents an declarative configuration of the Integration type for use
+// IntegrationApplyConfiguration represents a declarative configuration of the Integration type for use
 // with apply.
 type IntegrationApplyConfiguration struct {
-	DisableResolveMessage *bool            `json:"disableResolveMessage,omitempty"`
-	SecureFields          map[string]bool  `json:"SecureFields,omitempty"`
-	Settings              *json.RawMessage `json:"settings,omitempty"`
-	Type                  *string          `json:"type,omitempty"`
-	Uid                   *string          `json:"uid,omitempty"`
+	DisableResolveMessage *bool                  `json:"disableResolveMessage,omitempty"`
+	SecureFields          map[string]bool        `json:"secureFields,omitempty"`
+	Settings              *v0alpha1.Unstructured `json:"settings,omitempty"`
```

Comment:
@ryantxu this is an interesting change in the generated code

Reply:
Perhaps that happened because I forgot to regenerate code in https://github.com/grafana/grafana/pull/91430

Reply:
ah ok, that makes sense